File: abc289/abc289_a.cs
/*---------------------------------------------------



	   Welcome to my program!

	縲縲竏ｧ?ｼｿ竏ｧ        @x__0
	縲 ( 縲?ｽ･ﾏ会ｽ･)
	?ｼｿ(__縺､/?ｿ｣?ｿ｣?ｿ｣ /  MIT License
	縲縲?ｼｼ/縲縲縲縲 /  C# .Net 7.0
	縲縲縲縲?ｿ｣?ｿ｣?ｿ｣?ｿ｣?ｿ｣
		   Let's write Code!


---------------------------------------------------*/

using System;

namespace atcoder
{
	internal class Program
	{
		static void Main()
		{
			string? s = Console.ReadLine();
			char[] S = s.ToCharArray();
			for (int i = 0; i < s.Length; i++)
			{
				if (S[i] == '0') S[i] = '1';
				else S[i] = '0';
			}

			Console.WriteLine(S);
		}
	}
}

File: abc289/abc289_a.ts
/*------------------------------------------------------------



	   Welcome to my program!

	  ∧＿∧        AtCoder / Codeforces
	 ( 　･ω･)
	＿(__つ/￣￣￣ /  MIT License
	  ＼/　　　　 /   TypeScript  + Node.js v18.17.1
		￣￣￣￣￣
		   Let's write Code!


------------------------------------------------------------*/



//#region Import

import { readFileSync } from 'fs';
import * as mathjs from 'mathjs';
import * as std from 'tstl';

// #endregion

let N: number, M: number;
let H: number, W: number;
let S: string = "";
const space_split: boolean = true;

// Functions

let s: string;

function inputs(inp: Array<Array<string>>): void {
	s = inp[0][0];
}

function main(): void {
	let ans: string = "";
	for (let i = 0; i < s.length; i++) {
		if (s[i] === '0') ans += '1';
		else ans += '0';
	}

	console.log(ans);
}

// #region 入力

inputs(readFileSync("/dev/stdin", "utf-8").split("\n").map((line: string): Array<string> => {
	return line.split(" ");
}));

main();

// #endregion
